common/Configuring_CloudForms.adoc

```diff
@@ -0,0 +1,155 @@
+[[Configuring-cloudforms]]
+== Configuring {product-title}
+
+Although the {product-title} appliance comes configured to be integrated immediately into your environment, you can make some changes to its configuration.
+
+[NOTE]
+======
```

Comment:
By convention we limit block delimiters to four characters:

    ====

instead of

    ==============